src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -240,9 +241,13 @@ export default class CancerStudySelector extends QueryStoreComponent<ICancerStud
 				</SectionHeader>
 
 				<FlexRow className={styles.cancerStudySelectorBody}>
-					<div className={styles.cancerTypeListContainer}>
-						<this.CancerTypeList/>
-					</div>
+					<If condition={this.store.maxTreeDepth}>
+						<Then>

```

Comment:
We should try not rely on falsiness like this.  (this.store.maxTreeDepth > 0) is much more explicity/easier to understand for future devs.